src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedString = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '')
+    .reduce((prev, element) => {
+      const splittedStyle = element.split(':').map((el) => el.trim());
+
+      prev[splittedStyle[0]] = splittedStyle[1];
```

Comment:
Before assigning `splittedStyle[1]` to `prev[splittedStyle[0]]`, it's a good practice to check if both elements exist to avoid potential errors when dealing with malformed input strings.